src/modules/esl-utils/dom/scroll/test/utils.test.ts

```diff
@@ -0,0 +1,258 @@
+import {hasHorizontalScroll, hasVerticalScroll, isScrollLocked, lockScroll, unlockScroll} from '../utils';
+
+const $html = document.documentElement;
+const $body = document.body;
+
+afterEach(() => jest.clearAllMocks());
+
+const mockElHeight = (el: Element) => {
+  jest.spyOn(el, 'scrollHeight', 'get').mockImplementation(() => 100);
+  jest.spyOn(el, 'clientHeight', 'get').mockImplementation(() => 99);
+};
+
+const mockElWidth = (el: Element) => {
+  jest.spyOn(el, 'scrollWidth', 'get').mockImplementation(() => 100);
+  jest.spyOn(el, 'clientWidth', 'get').mockImplementation(() => 99);
+};
+
+describe('Function hasVerticalScroll', () => {
+  test('should be vertical scroll on default element', () => {
+    mockElHeight($html);
+    expect(hasVerticalScroll()).toBeTruthy();
+  });
+
+  test('should be vertical scroll on target element', () => {
+    const target = document.createElement('div');
+    mockElHeight(target);
+    expect(hasVerticalScroll(target)).toBeTruthy();
+  });
+});
+
+describe('Function hasHorizontalScroll', () => {
+  test('should be horizontal scroll on default element', () => {
+    mockElWidth($html);
+    expect(hasHorizontalScroll()).toBeTruthy();
+  });
+
+  test('should be horizontal scroll on target element', () => {
+    const target = document.createElement('div');
+    mockElWidth(target);
+    expect(hasHorizontalScroll(target)).toBeTruthy();
+  });
+});
+
+describe('Function isScrollLocked', () => {
+  const target = document.createElement('div');
+  target.style.overflow = 'auto';
+  describe('Locked Scroll', () => {
+    beforeAll(() => lockScroll(target));
+    test('lock attribute should be set', () => expect(target.hasAttribute('esl-scroll-lock')).toBeTruthy());
+    test('target should be locked', () => expect(isScrollLocked(target)).toBeTruthy());
```

Comment:
```suggestion
    test('target should be locked', () => expect(isScrollLocked(target)).toBe(true));
```
we are expecting exact boolean true